packages/playground/windows/.gitignore

```diff
@@ -33,6 +33,7 @@ ipch/
 [Bb]in
 [Dd]ebug*/
 [Rr]elease*/
+[Ww]inUI3*/
```

Comment:
Does this build in ship or debug? Might be better to add both to the matrix?

Reply:
This is Debug only for now (see the PR description for more details). Once we flip the switch we'll delete this configuration and make debug/release always use winui3.